packages/openapi-ts/src/index.spec.ts

```diff
@@ -19,6 +19,18 @@ describe('index', () => {
     });
   });
 
+  it.only('parses v3-transforms without issues', async () => {
+    await createClient({
+      client: '@hey-api/client-fetch',
+      dryRun: false,
+      input: './test/spec/v3-transforms.json',
+      output: './generated/v3/',
+      types: {
+        dates: false,
+      },
+    });
+  });
+
```

Comment:
I'll back this down to a dry-run over types+transform before approval, it's been my main dev tool since the debugger can hit breakpoints